### DISCUSSION TITLE
Add aspect ratio keeper mode (wysiwyg editor)

### DISCUSSION THREAD

**Discussion by ClementLaval** [directus/directus#11939] [Feature Requests]
![image](https://user-images.githubusercontent.com/82887830/156612607-dde7787d-d7d4-4087-8a0c-4feedc83696c.png)

Simple but smart.
![image](https://user-images.githubusercontent.com/82887830/156613164-42ef0a46-0fda-4e0f-916b-c5609baccb5c.png)

Add a aspect ratio calculator when user set a new width / height.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
